development/lib/run-command.js

```diff
@@ -96,7 +96,7 @@ async function runInShell(command, args, output) {
   const internalError = new Error('Internal');
   try {
     await new Promise((resolve, reject) => {
-      const childProcess = spawn(command, args);
+      const childProcess = spawn(command, args, { shell: true });
```

Comment:
This `runInShell` function now runs things in a shell.

This change was required because without it [shell expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Expansions.html) doesn't work, which the `runWebpack` function in `run-unit-tests.js` uses.